data/projects.json

```
@@ -138,6 +138,16 @@
         "level": "basic",
         "site": "",
         "tags": "car-game,c,c++,graphics-project"
+    },
+    {
+        "name": "Venessa-My-Personal-Assistant",
+        "display": "Personal-Assitant using python",
```

The display name could be 
```suggestion
        "display": "Personal Assitant using Python",
```